configure/os/CONFIG.Common.RTEMS-beatnik

```diff
@@ -8,14 +8,18 @@ EXE = .elf
 RTEMS_BSP = beatnik
 RTEMS_TARGET_CPU = powerpc
 GNU_TARGET = powerpc-rtems
-ARCH_DEP_CFLAGS += -DMY_DO_BOOTP=NULL
+# optimization trouble in postfix.c
+ARCH_DEP_CFLAGS += -DRTEMS_HAS_ALTIVEC
```

Comment:
Maybe just add `-mstrict-align` to CFLAGS and CXXFLAGS for this target?  see https://github.com/epics-base/epics-base/issues/29#issuecomment-508805177

Reply:
Is the testing from #29 sufficient to include -mstrict-align?  I can add this flag, but don't have a means to verify.